tardis/opacities/tests/test_opacity_state_numba.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from tardis.opacities.opacity_state import opacity_state_to_numba
+from tardis.opacities.opacity_solver import OpacitySolver
+import numpy.testing as npt
+import numpy as np
+
+
+@pytest.mark.parametrize(
+    "line_interaction_type,sliced",
+    [
+        ("scatter", False),
```

Comment:
Add a parameter that scatter = True to test the `macroatom = None` case in opacity_state.py line 42.

Reply:
Currently, because the initialization is dependent on the legacy plasma and is built from it in the opacity_state.from_legacy_plasma method, the line_interaction_type is not actually used by the solver in the current version.  This test will be added once the new plasma is in.